internal/pkg/flags/flag_to_value.go

```diff
@@ -5,13 +5,15 @@ import (
 	"time"
 
 	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
 )
 
 // Returns the flag's value as a string.
 // Returns "" if the flag is not set, if its value can not be converted to string, or if the flag does not exist.
-func FlagToStringValue(cmd *cobra.Command, flag string) string {
+func FlagToStringValue(cmd *cobra.Command, flag string, p *print.Printer) string {
 	value, err := cmd.Flags().GetString(flag)
 	if err != nil {
+		p.Debug(print.ErrorLevel, "convert flag to string value err: %v", err)
```

Comment:
```suggestion
		p.Debug(print.ErrorLevel, "convert flag to string value: %v", err)
```
Err in this case is redundant

Reply:
this comment applies to the rest of the PR